### ISSUE TITLE
6.4 Status pattern detector warnings

### ISSUE THREAD

**Issue by solojavier** [Adobe-Consulting-Services/acs-aem-commons#1511]
## Required Information

* [ ] AEM Version: 6.4 + aem-service-pkg-6.4.1
* [ ] ACS AEM Commons Version: 3.18.2
* [ ] Reproducible on Latest? yes

### Expected Behavior

No warnings shown on status pattern detector report.

### Actual Behavior

[Status pattern detector](http://localhost:4502/system/console/status-pattern-detector) contains several [Extraneous Content Usage](https://helpx.adobe.com/experience-manager/pattern-detection/ECU.html) and 1 [Non Backwards Compatible Code](https://helpx.adobe.com/experience-manager/pattern-detection/NBCC.html) for tika parser.

### Steps to Reproduce

* Install ACS AEM Commons 3.18.2
* Install [Pattern Detector](https://helpx.adobe.com/experience-manager/6-4/sites/deploying/using/pattern-detector.html)

**Comment by solojavier** [comment]
:(

**Comment by badvision** [comment]
No worries @solojavier -- I'll pin it to keep it open but it's not going to be the highest priority for getting 4.0.0 out the door.  We have some refactoring activities that are breaking changes (hence the bump to 4.x) that are getting finalized at the moment.

**Comment by badvision** [comment]
Could you attach the JSON version of the pattern detector results if you have it conveniently laying around?

**Comment by badvision** [comment]
[pattern detector report.xlsx](https://github.com/Adobe-Consulting-Services/acs-aem-commons/files/2665429/pattern.detector.report.xlsx)

Were there any other JSON reports?  That was only for the extraneous code usage (e.g. things which link to /lib or /apps directly) -- I converted your JSON to a more readable excel format in case it helps.

**Comment by solojavier** [comment]
You are right, there is another suspicion reported. I double checked the json report and I missed the first line when copying it over:

```
{"timestamp":"2018-12-11T15:37:58.235-05:00","suspicion":true,"pattern":{"code":"NBCC","type":"non.backwards.compatible.code","detective":"IncompatibleExceptionDetector","moreInfo":"https://www.adobe.com/go/aem6_NBCC"},"item":{"id":"16dc55647a8c286a7916ff301d80120dd8e8a353cf9885bbf67e07cafeb5c8f2","message":"The OSGi import org.apache.tika.parser.html, due to major package version change, referenced at bundle: com.adobe.acs.acs-aem-commons-bundle might no longer be resolvable"}}```

I've updated the previous comment with the complete report.

**Comment by badvision** [comment]
Thanks for that.  I updated the excel file as well so we have the full list.

**Comment by dinesh-cliffordchance** [comment]
Hi @joerghoh / @kwin

I have tried with the latest version of ACS-Commons but still am getting these kind of similar errors.

Is there any update on this issue? We are migrating from AEM 6.5.15 to AEMaaCS and we ran BPA , there we can see similar issues. Please provide suggestions to resolve the issues.
